public/components/cards/27.html

```diff
@@ -0,0 +1,48 @@
+<a class="block bg-[#fefefe] shadow-xl rounded-xl text-center p-5" href="">
+  <h5
+    class="text-3xl font-bold mb-1 text-transparent bg-clip-text bg-gradient-to-r from-blue-600 via-blue-500 to-blue-600"
+  >
+    Basic Plan
+  </h5>
+
+  <h6 class="text-slate-900 text-sm">
+    Perfect for private or small business websites!
+  </h6>
+
+  <div class="mt-3 text-slate-900">
+    <span class="text-2xl">$</span>
+    <h6 class="inline text-5xl font-bold">9</h6>
+    <span class="text-xs"> / month</span>
+    <p class="text-xs text-slate-700">Billed annually</p>
+  </div>
```

Comment:
I think we can make the structure a little neater here, let's try moving the "$9 / month" into a parent element like this.

```suggestion
<div>
  <h6>
    <span></span>
    <span></span>
    <span></span>
  </h6>
  
  <p><p>
</div>
```